WebRoot/admin/view.php

```diff
@@ -14,27 +14,35 @@
 		<nav>
 			<?php include 'Templates/navbar admin.php' ?>
 		</nav>
-		<main>
+		<main class="dashboard">
 			<?php if(!$user->isAdmin()): ?>
 				<div class="box info-error-box">
 					<h2>You are not logged in!</h2>
 				</div>
+			<?php else: ?>
+				<h1>Admin | Dashboard</h1>
+				<?php include 'Templates/search.php' ?>
+				<section>
+					<h2>Webapp General</h2>
+					<div class="box box-text">
+						<p>Version: 2.2</p>
+						<?php if ($user->isAdmin() && $commit != ''): ?>
+							<p>Last commit: <?php echo $commit ?></p>
+						<?php endif ?>
+						<p>Database revision: <?php echo $settings['DatabaseRevision'] ?></p>
+					</div>
+				</section>
+				<section>
+					<h2>Companies</h2>
+					<div class="box box-text">
+						<a href=""><p id="MsgTxt"></p></a>
+					</div>
+					<!-- <div class="information-error-box" style="display:none" id="MsgBox">
+					<div class="information-error-vertical"></div>
+					<a href=""><h2 id="MsgTxt"></h2></a>
+					</div> -->
```

Comment:
This is used by the JavaScript auto-scraper to display error messages if something happen. It's why the style is initially set to "display:none"
See `WebRoot/static/dhtml/admin.js`

Reply:
The JavaScript auto-scraper use the element with `id="MsgBox"` (to change `display:none`)...
So I don't know if `<div class="box box-text">` is supposed to be always displayed or not?

Reply:
I saw that the message shows: "scraping company name." I like seeing that message, even if now it doesn't work... 

`box-text` should be always displayed.